### PR TITLE
change brimdata/zq to brimdata/zed

### PR DESCRIPTION
**Pull request by nwt**
This reflects the change from brimdata/zq to brimdata/zed in brimdata/zed#2460.

### CODE REVIEW

File: .github/workflows/advance-zed.yml [hunk 1]
```diff
@@ -37,21 +37,21 @@ jobs:
           # that caused it to wait, reducing push failures down below.
           ref: ${{ github.ref }}
           token: ${{ secrets.ZQ_UPDATE_PAT }}
-      - run: git config --local user.email 'automation@brimsecurity.com'
+      - run: git config --local user.email 'automation@brimdataurity.com'
```
Comment (philrz) [inline .github/workflows/advance-zed.yml:40]:
```suggestion
      - run: git config --local user.email 'automation@brimsecurity.com'
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## v0.24.0
-* Update zq to [v0.29.0](https://github.com/brimdata/zq/releases/tag/v0.29.0)
+* Update Zed to [v0.29.0](https://github.com/brimdata/zed/releases/tag/v0.29.0)
```
Comment (philrz) [inline CHANGELOG.md:2]:
```suggestion
* Update zq to [v0.29.0](https://github.com/brimdata/zed/releases/tag/v0.29.0)
```

I find it a little odd to go through the CHANGELOG and revise history as if Zed was a thing back when it wasn't. So, the link change makes sense to me, but saying "Update Zed" in place of "Update zq" is questionable in my mind. If a user follows the PR links they're going to find a whole lot of Zed-not-being-mentioned which might seem confusing. I respect the spirit of the search & replace exercise though, and I doubt most users are going through super old release notes at this point, so I understand if you wanna still make these changes.

Regardless, I'm only making this comment at the first place I see it in this doc. So if you do roll back the "Zed" mention in the text part, you'd want to do it throughout that file.

**Review by philrz — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (nwt) [reply CHANGELOG.md:2]: I think you're right, so I'll change these back to "zq".

**Review by philrz — APPROVED** [review]

**Comment by nwt** [comment]
[Markdown link check](https://github.com/brimdata/brim/pull/1554/checks?check_run_id=2247406971) failed on https://wiki.centos.org/FAQ/General. That looks like a transient failure, so I'm going to ignore it and merge this.

**Pull request merged**